tcms/telemetry/templates/telemetry/testcase-breakdown.html

```diff
@@ -0,0 +1,71 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load static %}
+{% load comments %}
+{% load extra_filters %}
+{% load attachments_tags %}
+
+{% block title %}TestCase Breakdown{% endblock %}
+
+{% block contents %}
+
+    <div class="col-md-3">
+        Product:
+    </div>
+    <div class="col-md-3">
+        <select name="product" id="id_select_product" class="form-control selectpicker">
```

Comment:
Also need `<option value="">----------</option>` for Product and Test plan and the feature to query all.